src/composables/useInventoryCountImportItem.ts

```diff
@@ -0,0 +1,260 @@
+import { ref, computed, ComputedRef } from 'vue';
+import Dexie, { Table } from 'dexie';
+import { useProductMaster } from './useProductMaster';
+import { hasError } from '@hotwax/oms-api';
+import api from '@/api';
+import { wrap } from 'comlink';
+import type { InventorySyncWorker } from '@/workers/inventorySyncWorker';
+
+interface ScanEvent {
+    id?: number;
+    scanId: string;
+    importId: string;
+    locationSeqId?: string | null;
+    sku: string;
+    qty: number;
+    createdAt: string;
+    aggApplied: boolean;
+}
+
+interface InventoryCountImportItem {
+    inventoryCountImportId: string;
+    importItemSeqId: number;
+    sku: string;
+    productId: string | null;
+    aggApplied?: boolean;
+    productIdentifier: string;
+    locationSeqId?: string | null;
+    quantity: number;
+    lastScanAt: string;
+    syncedQty: number;
```

Comment:
```suggestion
    updatedAt: number;
```
Replace with last updated time
Shouldn't this be a number?